transcriptomics/transcriptomics_matrix_generator.py

```diff
@@ -45,23 +45,32 @@ def load_sample_info(self, json_file_path):
             self.treatments = [item['Treatment'] for item in sample_info]
             self.experiment_id = [item['ExperimentID'] for item in sample_info]
 
-    def download_gff(self, url, file_path):   
+    def download_gff(self, url, file_path):
         if not os.path.exists(file_path):
             subprocess.run(['wget', '-O', file_path, url], check=True)
+        self.process_gff(file_path)
+
+    def process_gff(self, file_path):
```

Comment:
Would rename to something like `write_gene_lengths`, since the function now saves the gene lengths in a csv.